CI is unhappy:
```
2021-02-25T11:36:06.5317581Z 	[error]: Build step io.quarkus.kubernetes.config.deployment.KubernetesConfigProcessor#handleAccessToSecrets threw an exception: java.lang.NullPointerException
2021-02-25T11:36:06.5322131Z 	at io.quarkus.kubernetes.config.deployment.KubernetesConfigProcessor.handleAccessToSecrets(KubernetesConfigProcessor.java:46)
2021-02-25T11:36:06.5325489Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-02-25T11:36:06.5327338Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-02-25T11:36:06.5329591Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-02-25T11:36:06.5331322Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2021-02-25T11:36:06.5332646Z 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:920)
2021-02-25T11:36:06.5334088Z 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
2021-02-25T11:36:06.5335674Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
2021-02-25T11:36:06.5337561Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2021-02-25T11:36:06.5338888Z 	at java.lang.Thread.run(Thread.java:748)
2021-02-25T11:36:06.5339879Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2021-02-25T11:36:06.5340560Z 
2021-02-25T11:36:06.5341279Z 	at io.quarkus.builder.Execution.run(Execution.java:116)
2021-02-25T11:36:06.5342936Z 	at io.quarkus.builder.BuildExecutionBuilder.execute(BuildExecutionBuilder.java:79)
2021-02-25T11:36:06.5344877Z 	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:153)
2021-02-25T11:36:06.5347762Z 	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:358)
2021-02-25T11:36:06.5349174Z 	... 45 more
2021-02-25T11:36:06.5349826Z Caused by: java.lang.NullPointerException
2021-02-25T11:36:06.5352848Z 	at io.quarkus.kubernetes.config.deployment.KubernetesConfigProcessor.handleAccessToSecrets(KubernetesConfigProcessor.java:46)
2021-02-25T11:36:06.5356206Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-02-25T11:36:06.5358647Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-02-25T11:36:06.5362560Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-02-25T11:36:06.5364326Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2021-02-25T11:36:06.5365686Z 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:920)
2021-02-25T11:36:06.5367148Z 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
2021-02-25T11:36:06.5368842Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
2021-02-25T11:36:06.5370764Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2021-02-25T11:36:06.5372091Z 	at java.lang.Thread.run(Thread.java:748)
2021-02-25T11:36:06.5373092Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
```

--- a/extensions/kubernetes-config/deployment/src/main/java/io/quarkus/kubernetes/config/deployment/KubernetesConfigProcessor.java
+++ b/extensions/kubernetes-config/deployment/src/main/java/io/quarkus/kubernetes/config/deployment/KubernetesConfigProcessor.java
@@ -43,7 +43,7 @@ public class KubernetesConfigProcessor {
                     new KubernetesRoleBuildItem.PolicyRule(
                             Collections.singletonList(""),
                             Collections.singletonList("secrets"),
-                            Arrays.asList("get", "list", "watch")))));
+                            Arrays.asList("get")))));
             roleBindingProducer.produce(new KubernetesRoleBindingBuildItem("view-secrets", false));
         }
 

